base/shell/explorer/lang/bg-BG.rc

```diff
@@ -196,3 +196,8 @@ BEGIN
     IDS_TASKBAR_STARTMENU_PROP_CAPTION "Свойства на задачната лента и на пусковия изборник"
     IDS_RESTORE_ALL "&Show Open Windows"
 END
+STRINGTABLE
```

Comment:
There should be an empty line before `STRINGTABLE` in each resource file.
```suggestion

STRINGTABLE
```
Apply this suggestion in other files as well.